Compiling error on Retropie

There is a compiling last version 0.222  error on Retropie in an raspberri pi4b.

Error is there is not files for arm in 3rdparty/asmjit/src/asmjit/

Here build log with error ...



Archiving libascii.a...
Compiling 3rdparty/asmjit/src/asmjit/core/arch.cpp...
../../../../../3rdparty/asmjit/src/asmjit/core/arch.cpp:32:12: fatal error: ../arm/armarchdata_p.h: No existe el fichero o el directorio
   #include "../arm/armarchdata_p.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [asmjit.make:1059: ../../../../libretro/obj/libretro/3rdparty/asmjit/src/asmjit/core/arch.o] Error 1
make[1]: *** [Makefile:82: asmjit] Error 2
make[1]: *** Se espera a que terminen otras tareas....
Compiling src/mame/drivers/caswin.cpp...
Archiving libaristocr.a...
make: *** [makefile:1385: linux] Error 2
Removing additional swap
/home/pi
Could not successfully build lr-mame - MAME emulator - MAME (current) port for libretro (/home/pi/RetroPie-Setup/tmp/build/lr-mame/mamearcade_libretro.so not found).

--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -62,6 +62,10 @@ endif
 #
 ###########################################################################
 
+ifeq (,$(findstring x86,$(LIBRETRO_CPU)))
+	FORCE_DRC_C_BACKEND = 1
+endif
+
 ifeq ($(platform), android)
 	#PTR64 = 0
 	# FIXME: Pass these to GENie and test...

--- a/scripts/src/3rdparty.lua
+++ b/scripts/src/3rdparty.lua
@@ -2224,6 +2224,9 @@ project "wdlfft"
 -- asmjit library
 --------------------------------------------------
 
+-- RETRO HACK BEGIN disable asmjit compilation if C backend used
+if not (_OPTIONS["FORCE_DRC_C_BACKEND"]=='1') then
+-- RETRO HACK END disable asmjit compilation if C backend used
 project "asmjit"
 	uuid "4539757c-6e99-4bae-b3d0-b342a7c49539"
 	kind "StaticLib"
@@ -2353,3 +2356,4 @@ project "asmjit"
 		MAME_DIR .. "3rdparty/asmjit/src/asmjit/x86/x86rapass.cpp",
 		MAME_DIR .. "3rdparty/asmjit/src/asmjit/x86/x86rapass_p.h",
 	}
+end

--- a/scripts/src/main.lua
+++ b/scripts/src/main.lua
@@ -347,8 +347,16 @@ end
 		"wdlfft",
 		ext_lib("jpeg"),
 		"7z",
-		"asmjit",
+-- RETRO HACK no asmjit if FORCE_DRC_C_BACKEND
+--		"asmjit",
 	}
+if not (_OPTIONS["FORCE_DRC_C_BACKEND"]=='1') then
+	links {
+		"asmjit"
+	}
+end
+-- RETRO HACK end
+
 if (STANDALONE~=true) then
 	links {
 		ext_lib("lua"),

